includes/frontend/blocks/class-crowdsignal-forms-nps-block.php

```diff
@@ -178,4 +178,18 @@ private function attributes() {
 			),
 		);
 	}
+
+	/**
+	 * Returns a nonce based on the NONCE.
+	 * The nonce creation is first attempted through crowdsignal_forms_nps_nonce filter.
+	 *
+	 * @since [next-version-number]
+	 */
+	private function create_nonce() {
+		$nonce = apply_filters( 'crowdsignal_forms_nps_nonce', self::NONCE );
+		if ( empty( $nonce ) ) {
```

Comment:
I think this is an error. If no filters are given than `$nonce = self::NONCE`; which kind of breaks.